sal_interfaces/CCCamera/CCCamera_Events.xml

```diff
@@ -853,4 +853,161 @@
         <Count>1</Count>
     </item>
 </SALEvent>
+<SALEvent>
+    <Subsystem>CCCamera</Subsystem>
+    <EFDB_Topic>CCCamera_logevent_focalPlaneSummaryInfo</EFDB_Topic>
+    <Description>Special event for use by header service. All of the information in this event is redundant (i,e. also supplied in
+     other settingsApplied events), but the information in this event is formatted for ease-of-use of the header service, specifically
+     information which is REB or Raft specific is repeated for each CCD. This event will be sent at the same time as other 
+     settingsApplied events</Description>
+    <item>
+        <EFDB_Name>ccdLocation</EFDB_Name>
+        <Description>CCD location in focal-plane (e.g. R22S01) (: delimited)</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>  
+    <item>
+        <EFDB_Name>raftBay</EFDB_Name>
+        <Description>The raft bay for each CCD listed above, of the form Rnn : delimited</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>ccdSlot</EFDB_Name>
+        <Description>The CCD slot for each CCD listed above, of the form SXn : delimited</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>1</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>rebSerialNumber</EFDB_Name>
+        <Description>REB hardware serial number (: delimited)</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>rebLSSTName</EFDB_Name>
+        <Description>REB LSST identifier (: delimited)</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>ccdLSSTName</EFDB_Name>
+        <Description>CCD LSST identifier (: delimited)</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>    
+    <item>
+        <EFDB_Name>raftLSSTName</EFDB_Name>
+        <Description>Raft LSST identifier (: delimited)</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>    
+    <item>
+        <EFDB_Name>ccdManSerNum</EFDB_Name>
+        <Description>Sensor manufacturer's serial number (: delimited)</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>ccdManufacturer</EFDB_Name>
+        <Description>Sensor manufacturer E2V or ITL (: delimited)</Description>
+        <IDL_Type>string</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>ccdType</EFDB_Name>
```

Comment:
Is this necessary since we have ccdManufacturer above?

Reply:
There is a lot of redundant information here, but I thought the goal was that the header service would not have to apply any logic.

Reply:
Right but ccdTypes is an array of numbers not a delimited string. Is it guaranteed to be in the same order as the ccdLocations?

Reply:
Yes. We treat them all as arrays, and they are all in the same order, but since SAL does not support String arrays they are converted to : delimited strings.